### ISSUE TITLE
Prevent items from griefing instead of disabling them at all time

### ISSUE THREAD

**Issue by Rubbertjuh** [KaiKikuchi/ForgeRestrictor#3]
Hello!

First of all, I thank you again for making this plugin. It's really nice!
I've got a question though.
Is it possible to just disable an item from doing harm to a claim instead of disabling the item in overall.

Right now I think it's only possible to disable the item entirely. Do you think it's possible to add a new feature so it only disables the item when trying to destroy a claim with it ? 

I'm sorry if this feature is already implemented. The documentation on this plugin isn't the best.
Maybe something to improve on? ^^ 

Thanks again!

**Comment by KaiKikuchi** [comment]
Hello! Sorry for the late reply.

I don't get why you need to block just few items. Can you provide a valid reason?

**Comment by Rubbertjuh** [comment]
Hello and thank you for your reply!

I don't want to block just a few items. 
I want to block certain items from actually damaging a claim instead of the item not being able to be used at all. Right now it's only possible to use the command to disable for example the red morning star which disables the use of it entirely.
But would it be possible to make it so the item doesn't get disabled entirely but just when the user clicks in a claim?

In short: Make the plugin disable use of the item just when clicked on a claim instead of disabling the item intirely. 
Do you understand what I mean? If not, please ask.

The reason for this is so the item doesn't become useless. People can still use the item to for example, mine but they can't use it to grief someones claim.

Can't wait to hear what you have to say about this!
Thanks for the help.

**Comment by KaiKikuchi** [comment]
That's what this plugin does. You can't use modded items inside claims and world guard regions if you're not trusted.

**Comment by Rubbertjuh** [comment]
Then there is something wrong. I'm using this plugin on my server but for some reason, when I add items with the command. 
For example the Red Morning Star
People are not able to use the right click at all. Not just on claims, but anywhere in the world.

I've got WorldGuard installed and I do have 1 __global__ region, which disabled PvP. Might it have to do something with that?

**Comment by KaiKikuchi** [comment]
Please post the exact command you're using. Also try to delete the ForgeRestrictor folder under plugins so you reset the plugin data.

If players can build inside the WorldGuard region, then it's fine. The plugin checks if the player can build on the target block.

**Comment by Rubbertjuh** [comment]
Hello,
I tested it some more and I did as you said. 
I removed the ForgeRestrictor folder so it reset the plugin data.
After this, I tried using the morning star ( Should work now as there is no data in the folder ) 
Then I used the following command to add the Red Morning star to the list:
/fres add aoe hand
    ( Held the morning star in my hand )
Item was succesfully added as I get the following message in both console and in game:
"Added 'PROJECTE_ITEMPE_RM_MORNING_STAR:* (100)' to aoe list

Then I tried to use the morning star again and it didn't allow me to use it. 
It didn't even allow me to break blocks  by just left clicking. The console was also being spammed with:
"[ForgeRestrictor] Confiscated ToastyNetworks items for 3 ticks. 

I also tried disabling the WorldGuard feature. Maybe that was the problem, but it wasn't.
If you need more information, or want to test it out on my server with full perms, tell me!

Thanks for helping.

**Comment by KaiKikuchi** [comment]
You didn't specify a range in your command. That means the item is blocked if there's a claim within 100 blocks. Try specifying a lower range (16?) and try it outside any claim.

**Comment by Rubbertjuh** [comment]
Thanks a lot for your fast reply,
This seems to have fixed the problem!

Question: Would it be possible to prevent certain items damaging other players when for example the PVP Deny flag in worldguard is active?
Some items from for example Quivermod bypass PvP Protection, If you want me to create a new Issue on this, please do tell.
Just wanted to know if this is a possibility.

**Comment by KaiKikuchi** [comment]
Unfortunately atm the WorldGuard integration allows pvp if you have build permission only.

**Comment by iKaneki-Ken** [comment]
@Rubbertjuh Are you any good with scripting/coding at all? If so, you can either make a same plugin to do this for you, or a simple scripting plugin that prevents PvP.

**Comment by KaiKikuchi** [comment]
Feel free to fork this repository and include your custom protection!

**Comment by Rubbertjuh** [comment]
@iKaneki-Ken unfortunately I don't have enough experience in coding :(
It would be awesome to have such plugin though! 
Do you have any experience in coding?

**Comment by iKaneki-Ken** [comment]
@Rubbertjuh Yes, I do, unfortunately though I'm rather tied up with school and work to do any freelance work. Albeit I do recommend a plugin called OtherDrops.

**Comment by KaiKikuchi** [comment]
@Rubbertjuh now using worldguard flags. This is a development build if you want to test it. Report any issue on a new issue. https://github.com/KaiKikuchi/ForgeRestrictor/releases/tag/v1.4-DEV-2
